automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -146,10 +148,21 @@ def satellite6_capsule_upgrade(admin_password=None):
         print('Wrong Capsule Version Provided to upgrade to. '
               'Provide one of 6.1, 6.2')
         sys.exit(1)
+    if not os.environ.get('CAPSULE_SUBSCRIPTION') and not os.environ.get(
+            'CAPSULE_AK'):
+        print('One of Env. Var CAPSULE_SUBSCRIPTION or CAPSULE_AK should be'
+              'defined!')
+        sys.exit(1)
     if admin_password is None:
         admin_password = os.environ.get('ADMIN_PASSWORD', 'changeme')
     # Setting Capsule61 Repos
     major_ver = distro_info()[1]
+    # Re-register Capsule for 6.2¬
+    if to_version == '6.2':
+        ak_name = os.environ.get('CAPSULE_SUBSCRIPTION')[2] if os.environ.get(
```

Comment:
This needs to be updated, you need to process the string because all os.environ.get will return either a string or None if the env var was defined or if not respectively.